zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -367,11 +378,16 @@ def get_cluster_services(
             'port': next(iter(obj['spec'].get('ports', [])), None),  # Assume first port is the used one.
 
             'service_name': service.name,
-            'service_namespace': obj['metadata']['namespace'],
+            'service_namespace': service_namespace,
             'service_type': service_type,
             'service_ports': obj['spec'].get('ports', None),  # Could be useful when multiple ports are exposed.
 
             'endpoints_count': endpoints_map.get(service.name, 0),
+
+            'deeplink1': '{}/#/clusters/{}/{}'.format(
```

Comment:
This should only be applied if application=spilo, pgview does not make sense to all services